Model/Sales/TrackTraceHolder.php

```diff
@@ -296,19 +296,17 @@ private function convertDataForCdCountry($magentoTrack)
         }
 
         $products = $this->getItemsCollectionByShipmentId($magentoTrack->getShipment()->getId());
-
         foreach ($products as $product) {
             $myParcelProduct = (new MyParcelCustomsItem())
                 ->setDescription($product['name'])
                 ->setAmount($product['qty'])
                 ->setWeight($this->getWeightTypeOfOption($product['weight']))
                 ->setItemValue($product['price'] * 100)
-                ->setClassification((int) $this->hsCode('catalog_product_entity_int', $product['product_id'], 'classification'))
-                ->setCountry('NL');
+                ->setClassification((int) $this->getAttributeValue('catalog_product_entity_int', $product['product_id'], 'classification'))
+                ->setCountry((string) $this->getCountryOfOrigin($product));
```

Comment:
Waarom gebruik je hier ook niet de getAttributeValue method dan is de getCountryOfOrigin helemaal niet nodig

Reply:
```suggestion
                ->setCountry((string) $this-> getAttributeValue('catalog_product_entity_varchar', $product['product_id'], 'country_of_origin'));
```